File: 3289-the-two-sneaky-numbers-of-digitville/3289-the-two-sneaky-numbers-of-digitville.js
var getSneakyNumbers = function(nums) {
    let seen = [];
    let dupes = [];
    for(let i = 0; i<nums.length; i++){
        if(seen.includes(nums[i])){
            dupes.push(nums[i])
        }else{
            seen.push(nums[i])
        }
    }
    return dupes
};

File: 3289-the-two-sneaky-numbers-of-digitville/3289-the-two-sneaky-numbers-of-digitville.cs
public class Solution {
    public int[] GetSneakyNumbers(int[] nums) {
        List<int> seen = new List<int>();
        List<int> dupe = new List<int>();
        for(int i = 0; i < nums.Length; i++){
            if(seen.Contains(nums[i]))
            {
                dupe.Add(nums[i]);
            }
            else
            {
                seen.Add(nums[i]);
            }
        }
        return dupe.ToArray();
    }
}